### PR TITLE
Gidk 23ghp7 tb test

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/tb-pl-bs/components/tb-pl-bs-grid-row.component.html [hunk 1]
```diff
@@ -1,13 +1,127 @@
...
+    <!-- <section class="row row-2 account " [ngClass]="{'isHidden': !account.isVisible }"> -->
+    <section class="row-2 account " style="overflow: visible;" *ngIf="account.isVisible || account.isCreated"
+        [hidden]="!account.isVisible" (dblclick)="accountInfo(account,$event)" (clickOutside)="hideModal()"
+        [exclude]="'.row-item'">
+
+        <div class="row row-item" style="height: 35px !important;"
+            *ngIf="account.name && (account.closingBalance?.amount !== 0 || account.openingBalance?.amount !== 0 || account.debitTotal || account.creditTotal)">
+            <div class="col-4 account no-select" [ngStyle]="{'padding-left': (padding+20)+'px'}">
+
+                <div style="padding: 0px;border-right: 0px;" [innerHTML]="account.name | lowercase | highlight:search">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/tb-pl-bs/components/tb-pl-bs-grid-row.component.html:48]: Remove inline style

File: apps/web-giddh/src/app/tb-pl-bs/components/tb-pl-bs-grid-row.component.html [hunk 2]
```diff
@@ -1,13 +1,127 @@
...
+
+    </div>
+</div>
+<ng-container *ngFor="let account of groupDetail.accounts;trackBy: trackByFn">
+    <!-- <section class="row row-2 account " [ngClass]="{'isHidden': !account.isVisible }"> -->
+    <section class="row-2 account " style="overflow: visible;" *ngIf="account.isVisible || account.isCreated"
+        [hidden]="!account.isVisible" (dblclick)="accountInfo(account,$event)" (clickOutside)="hideModal()"
+        [exclude]="'.row-item'">
+
+        <div class="row row-item" style="height: 35px !important;"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/tb-pl-bs/components/tb-pl-bs-grid-row.component.html:44]: remove inline style

File: apps/web-giddh/src/app/tb-pl-bs/components/tb/tb-grid/tb-grid.component.html [hunk 3]
```diff
@@ -84,11 +84,34 @@ <h1>{{localeData?.no_data_found}}</h1>
...
+                                            <amount-field [amount]="data$.creditTotal" [currencySymbol]="false" [currencyCode]="false">
+                                            </amount-field>
+                                        </span>
+                                    </th>
                                     <th width="16.6%;" class="text-right">
-                                        <span>{{ data$.closingBalance?.amount | giddhCurrency}}</span>
+                                        <span>
+                                            <!-- {{ data$.closingBalance?.amount | giddhCurrency}} -->
+                                            <span class="d-inline-flex">
+                                                <amount-field [amount]="data$.closingBalance?.amount " [currencySymbol]="false" [currencyCode]="false">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/tb-pl-bs/components/tb/tb-grid/tb-grid.component.html:111]:
```suggestion
                                                <amount-field [amount]="data$.closingBalance?.amount" [currencySymbol]="false" [currencyCode]="false">
```

File: apps/web-giddh/src/app/userDetails/components/subscriptions/subscriptions.component.html [hunk 4]
```diff
@@ -123,7 +123,13 @@ <h2 class="section-head pb-0">{{seletedUserPlans?.planDetails?.name}}</h2>
 
                                             <tr>
                                                 <th>Subscription Price</th>
-                                                <td>{{seletedUserPlans?.planDetails?.amount | number}}</td>
+                                                <td>
+                                                    <!-- {{seletedUserPlans?.planDetails?.amount | number}} -->
+                                                    <span class="d-inline-flex">
+                                                        <amount-field [amount]="seletedUserPlans?.planDetails?.amount" [currencySymbol]="false" [currencyCode]="false">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/userDetails/components/subscriptions/subscriptions.component.html:129]:
```suggestion
                                                        <amount-field [amount]="seletedUserPlans?.planDetails?.amount | number" [currencySymbol]="false" [currencyCode]="false">
```

Comment (ravinderwalkover) [inline apps/web-giddh/src/app/userDetails/components/subscriptions/subscriptions.component.html:129]: Add a new input in amount-field to apply/unapply giddhCurrency Pipe, like in this case it shouldn't be applied.

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/tb-pl-bs/components/tb-pl-bs-grid-row.component.html (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/tb-pl-bs/components/tb-pl-bs-grid-row.component.html:48]: Done

File: apps/web-giddh/src/app/tb-pl-bs/components/tb-pl-bs-grid-row.component.html (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/tb-pl-bs/components/tb-pl-bs-grid-row.component.html:44]: Done

File: apps/web-giddh/src/app/tb-pl-bs/components/tb/tb-grid/tb-grid.component.html (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/tb-pl-bs/components/tb/tb-grid/tb-grid.component.html:111]: Done

File: apps/web-giddh/src/app/userDetails/components/subscriptions/subscriptions.component.html (hunk 4 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/userDetails/components/subscriptions/subscriptions.component.html:129]: Done

Comment (shalinee-walkover) [reply apps/web-giddh/src/app/userDetails/components/subscriptions/subscriptions.component.html]: Done

File: apps/web-giddh/src/app/userDetails/components/subscriptions/subscriptions.component.html [hunk 5]
```diff
@@ -123,7 +123,13 @@ <h2 class="section-head pb-0">{{seletedUserPlans?.planDetails?.name}}</h2>
 
                                             <tr>
                                                 <th>Subscription Price</th>
-                                                <td>{{seletedUserPlans?.planDetails?.amount | number}}</td>
+                                                <td>
+                                                    <!-- {{seletedUserPlans?.planDetails?.amount | number}} -->
+                                                    <span class="d-inline-flex">
+                                                        <amount-field [amount]="seletedUserPlans?.planDetails?.amount |  number}" [currencySymbol]="false" [currencyCode]="false" [useGiddhCurrencyPipe]="false">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/userDetails/components/subscriptions/subscriptions.component.html:129]:
```suggestion
                                                        <amount-field [amount]="(seletedUserPlans?.planDetails?.amount |  number)" [currencySymbol]="false" [currencyCode]="false" [useGiddhCurrencyPipe]="false">
```

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/userDetails/components/subscriptions/subscriptions.component.html (hunk 5 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/userDetails/components/subscriptions/subscriptions.component.html]: Done

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**